.changeset/serious-crews-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+[wrangler] fix: constellation command help
```

Comment:
Can you give an example of the previous wrong output, and the new correct output?